wrapper.html

```diff
@@ -52,6 +52,22 @@
 		})(window, document, 'script', 'dataLayer', 'GTM-MNNGDHL');
 	</script>
 	<!-- End Google Tag Manager -->
+	<!-- Start VWO Async SmartCode -->

```

Comment:
@sukhada this is in master as well?

Reply:
yep: https://github.com/350org/ak_intl_v3/commit/16a875a13aa1cd81257b7947a206451e08b6b6ca